sql/parse/util.go

```diff
@@ -516,3 +516,65 @@ func maybeList(opening, separator, closing rune, list *[]string) parseFunc {
 		}
 	}
 }
+
+// A qualifiedName represents an identifier of type "db_name.table_name"
+type qualifiedName struct {
+	qualifier string
+	name      string
+}
+
+// readQualifiedIdentifierList reads a comma-separated list of qualifiedNames.
+// Any number of spaces between the qualified names are accepted. The qualifier
+// may be empty, in which case the period is optional.
+// An example of a correctly formed list is:
+// "my_db.myview, db_2.mytable ,   aTable"
```

Comment:
micro-nit: two spaces after the second comma

Reply:
It is intended. The parser accepts any number of spaces between the names and the commas, as said above, in the same comment.

Reply:
Ah, ok!